items.json

```diff
@@ -3320,9 +3320,9 @@
     "name": "Battanian Mountain Blade",
     "culture": "Battania",
     "type": "TwoHandedWeapon",

```

Comment:
any weapon that have one handed mode and two handed mode will be displayed in the onehanded category. If so , you should rename it with cavalry in the begining of the name to make clear that it's a bad melee weapon but a good cavalry sword

Reply:
removed 1h modes from most weps